src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -334,4 +304,13 @@ function step!(integrator::PairedExplicitRK3Integrator)
         terminate!(integrator)
     end
 end
+
+function Base.resize!(integrator::PairedExplicitRK3Integrator, new_size)
+    resize!(integrator.u, new_size)
+    resize!(integrator.du, new_size)
+    resize!(integrator.u_tmp, new_size)
+
+    resize!(integrator.k1, new_size)
+    resize!(integrator.kS1, new_size)
+end
```

Comment:
Codecov reports that this is not covered. Is this because there is no elixir with AMR for the PERK 3 method? If yes, maybe we should add one (potentially in a separate PR)?

Reply:
Hm true. I guess one could stitch together a test just for this.